tests/test-tls13.conf

```diff
@@ -9,10 +9,12 @@
 # server TLSv1.3 TLS13-AES128-GCM-SHA256
 -v 4
 -l TLS13-AES128-GCM-SHA256
+-6
```

Comment:
I just picked one test for TLS 1.3 and one for 1.2, but let me know if I should add more.